src/avs/AVSDetailsHeader.jsx

```diff
@@ -0,0 +1,152 @@
+import { Link, Image, Button, Skeleton } from '@nextui-org/react';
+import React, { useCallback } from 'react';
+import { useMutativeReducer } from 'use-mutative';
+import { reduceState } from '../shared/helpers';
+
+const formatEtherscanLink = address => {
+  return `https://etherscan.io/address/${address}`;
+};
+
+export default function AVSDetailsHeader({ avs }) {
+  const [state, dispatch] = useMutativeReducer(reduceState, {
+    isCopied: false,
+    copyTimeout: null
+  });
+
+  const handleCopy = useCallback(value => {
+    if ('clipboard' in navigator) {
+      navigator.clipboard.writeText(value).catch();
+
+      if (state.copyTimeout) {
+        clearTimeout(state.copyTimeout);
+      }
+
+      dispatch({
+        isCopied: true,
+        copyTimeout: setTimeout(() => dispatch({ isCopied: false }), 2000)
+      });
+    }
+  }, []);
+
+  return (
+    <div className="bg-content1 border border-outline rounded-lg w-full p-4 break-words">
+      <div className="flex items-center">
+        <AVSLogo avs={avs} />
+        <span className="ml-2 text-foreground-1 text-3xl font-display font-medium">
+          {avs.metadata?.name ?? 'N/A'}
+        </span>
+        {/*TODO: implement ranking when coming from list view & accessing directly avs*/}
+        <div className="text-xs ml-2 p-1 rounded-md bg-foreground-2 text-content1">
+          # 1
+        </div>
+      </div>
+      <div className="my-4 text-xs text-foreground-1 break-words">
+        {avs.metadata?.description ?? 'N/A'}
+      </div>
+
+      {avs.metadata?.website && (
+        <Link
+          href={avs.metadata.website}
+          target="_blank"
+          rel="noreferrer"
+          className="text-xs text-secondary"
+        >
+          {avs.metadata.website}
+        </Link>
+      )}
+
+      <div className="flex">
+        <div className="flex items-center">
+          <Link
+            href={formatEtherscanLink(avs.address)}
+            target="blank"
```

Comment:
That function seems redundant:

```suggestion
            href={`https://etherscan.io/address/${avs.address}`}
            target="_blank"
```